pkgs/development/python-modules/scalene/default.nix

```diff
@@ -37,12 +40,20 @@ buildPythonPackage rec {
     cloudpickle
     jinja2
     psutil
-    pynvml
     rich
+    pynvml # to be removed on next release
+  ] ++ lib.optionals stdenv.hostPlatform.isLinux [
+    nvidia-ml-py # only avilable on linux. Scalene only loads this there.
   ];
 
   __darwinAllowLocalNetworking = true;
 
+  # NOTE: Remove on next release
+  patches = [
```

Comment:
Let's use [`pythonRemoveDeps`](https://nixos.org/manual/nixpkgs/unstable/#using-pythonrelaxdepshook) if possible rather than patching

Reply:
Thank you! I hadn't seen that before.

Done.